src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let stateClone = { ...state };
```

Comment:
Consider using const instead of let without extra need

Reply:
I'm creating `stateClone` intentionally as a copy of the initial `state` so I can modify it in the code below.